### DISCUSSION TITLE
How to use a Dropdown with Badge.Ribbon?

### DISCUSSION THREAD

**Discussion by neongreen** [ant-design/ant-design#36492] [Q&A]
I have a ribbon and I want to open a menu when clicking on that ribbon. Is it possible?

<img width="439" alt="image" src="https://user-images.githubusercontent.com/1523306/178616467-5503f46e-d6e3-4dc7-b0f6-201f8a098a7e.png">

**Comment by siam88** [comment]
The design you showes above, You can put those badges outside ta menu but This might not be the actual solution further for work done I just write a demo code . Hope it works for you

code : [demo code ](https://stackblitz.com/edit/react-2xvxdr?file=demo.js)
